core/trino-parser/src/main/java/io/trino/sql/parser/AstBuilder.java

```diff
@@ -2251,6 +2278,18 @@ public Node visitFunctionCall(SqlBaseParser.FunctionCallContext context)
             return new TryExpression(getLocation(context), (Expression) visit(getOnlyElement(context.expression())));
         }
 
+        if (name.toString().equalsIgnoreCase("trim") || name.toString().equalsIgnoreCase("ltrim") || name.toString().equalsIgnoreCase("rtrim")) {
```

Comment:
`ltrim` and `rtrim` should be parsed by the same rule as `trim`. Since we don't want to make them reserved words, the rule should probably be moved above the rule for `functionCall`.

Reply:
Do you mean adding `ltrim` and `rtrim` to SqlBase.g4?

Reply:
Actually, let's leave them where they are.